### DISCUSSION TITLE
Version 5.1.1.0

### DISCUSSION THREAD

**Discussion by rsjaffe** [rsjaffe/MIDI2LR#541] [Announcements]
## New features
  * Quick develop button commands for Exposure, Contrast, Highlights, Shadows, Whites, Blacks, Clarity, Vibrance and Saturation. See [Commands Page](https://github.com/rsjaffe/MIDI2LR/wiki/Commands) under *Quick develop* for details. Dehaze and Texture quick develop actions are *not* currently possible with Lightroom as Adobe seems to have overlooked them. A bug report has been filed.
## Bug fixes
  * Show spot removal command was not working, fixed.
  * Fixes crash on shutdown in MacOS.

<hr /><em>This discussion was created from the release <a href='https://github.com/rsjaffe/MIDI2LR/releases/tag/v5.1.1.0'>Version 5.1.1.0</a>.</em>

**Comment by derherrseifert** [comment]
Great Update! No crash at closing LR anymore 🥇
